### PR TITLE
feat(toolbox): add conda package definition

### PR DESCRIPTION
**Pull request by safl**
This is done to meet a request for xNVMe as a Conda package, see https://github.com/xnvme/xnvme/issues/506 for tracking of additional features to add to the content of the Conda package.

- [x] Switch Conda recipes to off-tree: https://github.com/xnvme/xnvme-conda

### CODE REVIEW

File: .github/workflows/verify.yml [hunk 1]
```diff
@@ -1255,3 +1255,63 @@ jobs:
...
+        cd xnvme/toolbox/conda
+        conda activate xnvme_conda_build
+        conda-build . --output-folder /tmp/conda_pkg
+        find /tmp/conda_pkg
+
+    - name: Find the built conda package
+      id: find-package
+      run: |
+        PACKAGE_PATH=$(find /tmp/conda_pkg -name "*.tar.bz2" | head -n 1)
+        echo "PACKAGE_PATH=$PACKAGE_PATH" >> $GITHUB_ENV
```
Comment (karlowich) [inline .github/workflows/verify.yml:1310]: Do we need this step?

**Review by karlowich — COMMENTED** [review]
Just a single comment

File: .github/workflows/verify.yml (hunk 1 above)
Comment (mbrsamsung) [reply .github/workflows/verify.yml:1310]: What do you mean?

Comment (safl) [reply .github/workflows/verify.yml:1310]: > Do we need this step?

Yes, to produce a path for "upload-artifact", making the built-artifact available.

**Comment by karlowich** [comment]
>  I'll move the Conda-recipes into a repository of its own
I agree with this approach

File: .github/workflows/verify.yml (hunk 1 above)
Comment (safl) [reply .github/workflows/verify.yml:1310]: > Maybe I'm not reading the code correctly - but it seems to set an environment variable that we never use

hah, you are right, I had forgotten that I went with the entire build folder to get the auxiliary artifacts as well. So, nice catch!

File: docs/contributing/packaging/index.rst [hunk 2]
```diff
@@ -19,6 +19,10 @@ see the :xref-github-xnvme-issues:`GitHUB Issue Tracker<389>`.
 
   - See: https://formulae.brew.sh/formula/xnvme
 
+* Conda
+
+  - See: h
```
Comment (karlowich) [inline docs/contributing/packaging/index.rst:24]: Something is missing here

**Comment by safl** [comment]
Switching this back to DRAFT as the conda recipe itself is now in being staged for conda-forge:
https://github.com/conda-forge/staged-recipes/pull/26854

Thus, this PR here will probably change into just being maintenance documentation.

**Comment by safl** [comment]
Closing this, since it might be a while and by then I expect the Documentation overhaul will be finalized and thus simpler to update the documentation then. Status / progress on the Conda packaging is tracked here:

- https://github.com/xnvme/xnvme/issues/389
- https://github.com/xnvme/xnvme/issues/506

**Pull request closed without merging**